README.md

```diff
@@ -1,33 +1,30 @@
 # Gameplay
 
-Issue tracker for Unvanquished gameplay related feedback, opinions, proposals
-and new ideas.
+Issue tracker for Unvanquished gameplay related feedback, problems and new ideas.
 
 To have a fruitful discussion, every issue here **must** fall clearly into one
 of the following categories:
 
-## Feedback
+## Experience & Feedback
 
 You can talk freely about how you felt during a game of Unvanquished or when
 playing in general. You can let us know what was frustrating or enjoyable to
 you but please make it clear that these are your feelings and refrain from
-including any opinions, proposals or ideas in such an issue. This way we can
-just accept your feelings as a fact and are free to pursue any number of
-solutions or improvements in another issue. If your feedback suggests that a
-particular thing needs to change in a particular way, we will close it.
+including any proposals or ideas in such an issue.
 
-Unlike opinions, feedback may not be answered with solution proposals but it
+Unlike `Problems`, `Experience & Feedback` are only for documentation purpose and not answered with solution proposals but it
 can help developers and players to come up with or agree with opinions about
-the current state of the game. Because it is inherently true and cannot be
-rejected, pure feedback can be very helpful and influencial in the long run.
+the current state of the game. 
+
+`Experience & Feedback` issues are closed 60 days after last activity. 
```

Comment:
I'm not sure if we should set it in stone like this because 60 days aren't much, experiences can stay valid for much longer. We could close once we believe the game has changed enough that the experience could be different.

Reply:
It depends on how we treat such issues. They are just a snapshot of the current game and player at that moment.
I think if after a longer time of silence there will much meaningful discussion happen to a single experience. Closing the issue wouldn't state a judgement about the validity of an experience (witch would be hard to do) but end the discussion about it. Any future talks should happen in related problem issues.

And if another player has similar experience they can create a new issue.

Reply:
Alright then.